scripts/views/searchView.js

```diff
@@ -0,0 +1,147 @@
+            searchView.currentCar.id = year.id;
+          }
+        });
+        if (!$('.maintenance-Item')) {
+          searchView.showCarPhoto();
+        } else {
+          $('.maintenance-Item').remove();
+          searchView.showCarPhoto();
+        }
+        $('#push-to-garage').show();
+      }
+    });
+  };
+
+  searchView.addCar = function() {
+    var userEmail = localStorage.getItem('currentUser');
+    $.get('/addCar', {currentCar: searchView.currentCar, email: userEmail});
+  };
+
+  var render = function(carData) {
+    var garageTemplate2 = Handlebars.compile($('#maintenance-template').html());
+
+    return garageTemplate2(carData);
+  };
+
+  searchView.showCarMaintenance = function(data) {
+    searchView.searchedCarMaintenance = [];
+    data.actionHolder.map(function(maintenanceItem, index) {
+      searchView.searchedCarMaintenance.push(maintenanceItem);
+      if (index % 10 === 0) {
+        var listItem = '<li class="maintenance-Item"><strong> ITEM: </strong> ' + searchView.searchedCarMaintenance[index].item + '<br><strong> ACTION: </strong> ' + searchView.searchedCarMaintenance[index].action + '<br><strong> DESCRIPTION:</strong> ' + searchView.searchedCarMaintenance[index].itemDescription + '<br><strong> INTERVAL MILEAGE: </strong> ' + searchView.searchedCarMaintenance[index].intervalMileage + '<br></li><p>';
+        // console.log(searchView.searchedCarMaintenance[index]);
+        $('#maintenance').append(listItem);
+      };
+    });
+  };
+
+  searchView.showCarPhoto = function() {
+    if ($('#car-container')) {
+      $('#car-container').remove();
+    }
+    var carPhoto = '<div id="car-container"><p>Search Results:<div id="car-photo"> <img src="http://media.ed.edmunds-media.com/audi/s7/2013/oem/2013_audi_s7_sedan_prestige_fq_oem_6_1600.jpg"></div><p id="results-text">' + searchView.currentCar.make + ' ' + searchView.currentCar.model + ' - ' + searchView.currentCar.year + '</p></div>';
+    $('#search').append(carPhoto);
+  };
+
+  searchView.createModelFilter = function(makeVal) {
+    searchTool.AllCars[0].makes.filter(function(make, indx) {
+      if (make.name === makeVal) {
+        searchView.index = indx;
+        searchTool.AllCars[0].makes[indx].models.map(function(model) {
+          var optionTag = '<option class="model-filter" value="' + model.name + '">' + model.name + '</option>';
+          $('#model-filter').append(optionTag).show();
+        });
+      }
+    });
+  };
+
+  searchView.createYearFilter = function(modelVal) {
+    searchTool.AllCars[0].makes[searchView.index].models.filter(function(model, indx) {
+      if (model.name === modelVal) {
+        searchView.modelIndex = indx;
+        searchTool.AllCars[0].makes[searchView.index].models[searchView.modelIndex].years.map(function(year) {
+          var optionTag = '<option class="year-filter" value="' + year.year + '">' + year.year + '</option>';
+          $('#year-filter').append(optionTag).show();
+        });
+
```

Comment:
There's some odd white space in the JS files. It's a minor detail, but from a readability standpoint it feels choppy at times and doesn't show an attention to fit and finish.